common/test/acceptance/pages/lms/learner_profile.py

```diff
@@ -43,7 +43,10 @@ def is_browser_on_page(self):
         """
         Check if browser is showing correct page.
         """
-        return self.q(css='body.view-profile .account-settings-container').present
+        return all([
+            self.q(css='body.view-profile .account-settings-container').present,
+            not self.q(css='ui-loading-indicator').visible
```

Comment:
Ok. Thanks a lot for taking a look. I think what you describe is what happens....I may have looked at this previously, myself.